### PR TITLE
update docs to reflect that shard keyranges can be > 1 byte

### PR DESCRIPTION
**Pull request by demmer**
As discussed in the slack channel, I mistakenly that the shard keys needed to map to a single byte and hence a limit of 256 shards.

This PR updates the examples in the docs to show that key ranges can span more than one byte.

### CODE REVIEW

File: docs/user-guide/sharding.html [hunk 1]
```diff
@@ -392,14 +393,16 @@ <h3 id="shard-names-in-range-based-keyspaces">Shard Names in Range-Based Keyspac
 beginning with [ 0x80 ] and ending, noninclusively, with [ 0xc0], then
 the shard&#39;s name is <code>80-c0</code>.</p>
 
-<p>Using this naming convention, the following four shards would be a valid
-full partition:</p>
+<p>Using this naming convention, the following six shards would be a valid
```
Comment (michael-berlin) [inline docs/user-guide/sharding.html]: How about keeping the original example as a simple case and mention your advanced example right after that?

People are probably looking for a simple example first: A setup with 2 or 4 shards.

You can introduce your advanced example by stressing that the user can use more byte values to further divide the keyrange.

One other note: You're introducing the term "uneven". How about explaining "even" first right after the simple example?

Something like this:

> This is an example for a partition with even shards i.e. equally sized key ranges.

And then you can say that uneven shards are supported and more byte values as well :)

**Comment by demmer** [comment]
I signed it!

**Comment by demmer** [comment]
@michael-berlin Thanks for the feedback -- this was on the back burner for a bit while I got the CLA established for Slack, but now that's all set.

I pushed a new commit that should address your concerns and this time I edited `Sharding.md`. It wasn't immediately obvious to me how to regenerate sharding.html so I didn't do that yet but am happy to do so.

**Comment by michael-berlin** [comment]
Thanks :)

It looks like the new commit isn't pushed yet?

re: the re-generation of the .html file. I will take care of this.

**Comment by demmer** [comment]
@michael-berlin My bad... fat-fingered the push command and I see it never went through.
642261a781da6cdc05ce393a8ea44c59850a25a4 is now pushed.

**Comment by michael-berlin** [comment]
LGTM

Thank you for the contribution :)

I ran `./publish-site.sh --docker` to re-generate the HTML files and pushed it to your branch. (By default, contributors give maintainers write access to pull request branches on GitHub.)

Merging now.

**Pull request merged**